### ISSUE TITLE
Fatal error message when reactivating plugin

### ISSUE THREAD

**Issue by markhowellsmead** [SayHelloGmbH/git-installer#49]
The following fatal error message was thrown when reactivating Git Installer, when a pre-existing configuration was available.

WordPress 6.3 and PHP 8.1.

```
[16-Aug-2023 13:49:57 UTC] PHP Fatal error: Uncaught TypeError: class@anonymous::fetchFileContent(): Return value must be of type string, null returned in ***WEBROOT***/wp-content/plugins/git-installer/src/Package/provider/Bitbucket.php:195
Stack trace:
#0 ***WEBROOT***/wp-content/plugins/git-installer/src/Package/GitPackages.php(382): class@anonymous->fetchFileContent('https://api.bit...')
#1 ***WEBROOT***/wp-content/plugins/git-installer/src/Package/Updater.php(124): SayHello\GitInstaller\Package\GitPackages->loadNewPackageHeaders('cubetech-wpthem...')
#2 ***WEBROOT***/wp-content/plugins/git-installer/src/Package/Updater.php(180): SayHello\GitInstaller\Package\Updater->getNewPackageHeaders('cubetech-wpthem...')
#3 ***WEBROOT***/wp-includes/class-wp-hook.php(312): SayHello\GitInstaller\Package\Updater->updateThemes(Object(stdClass))
#4 ***WEBROOT***/wp-includes/plugin.php(205): WP_Hook->apply_filters(Object(stdClass), Array)
#5 ***WEBROOT***/wp-includes/option.php(2021): apply_filters('site_transient_...', Object(stdClass), 'update_themes')
#6 ***WEBROOT***/wp-includes/update.php(913): get_site_transient('update_themes')
#7 ***WEBROOT***/wp-admin/menu.php(33): wp_get_update_data()
#8 ***WEBROOT***/wp-admin/admin.php(158): require('/home/tirapapa/...')
#9 ***WEBROOT***/wp-admin/plugins.php(10): require_once('/home/tirapapa/...')
#10 {main}
thrown in ***WEBROOT***/wp-content/plugins/git-installer/src/Package/provider/Bitbucket.php on line 195
```

**Comment by markhowellsmead** [comment]
The linked repository at Bitbucket, which was set up previously, was renamed and so the URL / slug which is stored in WordPress is no longer current.

**Comment by nico-martin** [comment]
Solved with PR #51
